### DISCUSSION TITLE
Web3Auth react native sdk causes app to crash in android

### DISCUSSION THREAD

**Discussion by Haris565** [Web3Auth/Web3Auth#529] [Help]
Hi I'm trying to add web3auth in react native app but whenever i install  @toruslabs/react-native-web-browser this dependency it cause build to failed with this error **Execution failed for task ':toruslabs_react-native-web-browser:compileDebugKotlin'. this error**  but working fine in IOS

**Comment by shahbaz17** [comment]
Hey @Haris565 
Are you following https://web3auth.io/docs/sdk/react-native/ ?

Also, can you share some screenshots of the build failing?

**Comment by yashovardhan** [comment]
@Haris565 was this resolved?

**Comment by iovreche** [comment]
I get the same error with Android SDK 2.4.0 
In Android SDK 2.5.0, it fails requiring API 32 instead of 21

Docs are a mess and examples are only made in Expo... no serious person in React Native will ever use Expo, you need to eject sooner or later.

**Reply by himanshuchawla009** [reply]
Hi we are working on updating react native docs, @yashovardhan is spearheading the effort and he will soon announce it on github discussions as soon as they will be done.

**Reply by shahbaz17** [reply]
The fix is out in the latest release: [v3.2.1](https://github.com/Web3Auth/web3auth-react-native-sdk)

Docs: https://web3auth.io/docs/sdk/react-native/
Example Apps: https://github.com/Web3Auth/examples/tree/main/react-native
